maven-plugin/samples/pom.xml

```diff
@@ -103,5 +93,16 @@
 				</configuration>
 			</plugin>
 		</plugins>
+		<pluginManagement>
+			<plugins>
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-deploy-plugin</artifactId>
```

Comment:
Make this a property <maven.deploy.skip>true..

Reply:
Done